.github/workflows/reusable-jira-issue_to_jira.yml

```
@@ -30,6 +30,14 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 20
     steps:
+    - name: Set Jira fields
+      id: set_jira_fields
+      run: |
+        if [[ -z "${{ inputs.components }}" ]]; then
+          echo 'jira_fields={}' >> $GITHUB_OUTPUT
+        else
+          echo 'jira_fields={"components": ${{ inputs.components }} }' >> $GITHUB_OUTPUT
```

ho, maybe an opportunity to simply then.
From the repos, just pass the list of components name: `components: ["geoh5py"]`
and here:
```suggestion     
          components_json=$(echo '${{ inputs.components }}' | jq -c '{"components": [.[] | {name: .}]}')
          echo "jira_fields=$components_json" >> $GITHUB_OUTPUT
```

---

or, at once with jq, with on more if/else
```
    components_json=$(echo '${{ inputs.components }}' | jq -c '{"components": (if . == "" then [] else [.[] | {name: .}] end)}')
```

---

I thought about that, but there may be a future case where we want to pass something in addition to "components" -- the "fields" param can be used to pass other things too (labels, "Customer email", etc).

---

nah, that one does not work has jq expects some json input. Keep bash if/else...

---

ok, no need to change